addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -712,13 +1026,50 @@ function ThreadFactory({ Entity }) {
             default: false,
         }),
         name: attr(),
+        /**
+         * Ordered typing members on this thread, excluding the current partner.
+         */
+        orderedOtherTypingMembers: many2many('Partner', {
+            compute: '_computeOrderedOtherTypingMembers',
+            dependencies: ['orderedTypingMembers'],
+        }),
+        /**
+         * Ordered typing members on this thread. Lower index means this member
+         * is currently typing for the longest time. This list includes current
+         * partner as typer.
+         */
+        orderedTypingMembers: many2many('Partner', {
+            compute: '_computeOrderedTypingMembers',
+            dependencies: [
+                'orderedTypingMemberLocalIds',
+                'typingMembers',
+            ],
+        }),
+        /**
+         * Technical attribute to manage ordered list of typing members.
+         */
+        orderedTypingMemberLocalIds: attr({
+            default: [],
+        }),
         originThreadAttachments: one2many('Attachment', {
             inverse: 'originThread',
         }),
         public: attr(),
         seen_message_id: attr(),
         seen_partners_info: attr(),
+        /**
+         * Members that are currently typing something in the composer of this
+         * thread, including current partner.
+         */
         typingMembers: many2many('Partner'),
+        /**
+         * Text that represents the status on this thread about typing members.
+         */
+        typingStatusText: attr({
+            compute: '_computeTypingStatusText',
+            default: _lt("No one is typing."),
```

Comment:
Do we need this default? Is it ever displayed?

Reply:
Not needed, just a placeholder for the sake of a placeholder ATM.

Reply:
Similar to another comment, not especially found of "dead code", but not a big deal either.

Reply:
Unresolving this, you need to remove this value to fix the crash in ellipsis title prop validation -> no default, or empty string